src/main/java/com/projectg/geyserupdater/bungee/util/GeyserBungeeDownloader.java

```diff
@@ -82,10 +82,11 @@ private static boolean downloadGeyser() {
      * Attempt to restart the server
      */
     private static void restartServer() {
-        logger.warn("The server will be restarting in 10 seconds!");
+        long restartTime = plugin.getConfig().getLong("Auto-Restart-Timer");
+        logger.warn("The server will be restarting in %d seconds!".formatted(restartTime));
         for (ProxiedPlayer player : plugin.getProxy().getPlayers()) {
-            player.sendMessage(new TextComponent(ChatColor.translateAlternateColorCodes('&', plugin.getConfig().getString("Restart-Message-Players"))));
+            player.sendMessage(new TextComponent(ChatColor.translateAlternateColorCodes('&', plugin.getConfig().getString("Restart-Message-Players").formatted(restartTime))));
         }
```

Comment:
Can we extract the component creation to outside the loop - same with the other platforms